aws/cloudformation/metaflow-cfn-template.yml

```diff
@@ -1,5 +1,5 @@
 AWSTemplateFormatVersion: '2010-09-09'
-Description: Stack for complete deployment of Metaflow (v2.0.0)
+Description: Stack for complete deployment of Metaflow (v2.2.5)
```

Comment:
Can we remove `(v2.2.5)` and instead just add `last-updated-date` or something similar to track template versions.

Reply:
Yup, all set.